sources/academy/webscraping/advanced_web_scraping/crawling/crawling-sitemaps.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Crawling sitemaps
+description: Learn how to extract all of a website's listings even if they limit the number of results pages. See code examples for setting up your scraper.
+menuWeight: 2
```

Comment:
Is this something that is custom created by Apify? I haven't seen this anywhere else